css/index.css

```diff
@@ -1,25 +1,47 @@
-.textInput {
+.urlInput {
   width: 100%;
-  padding: 12px 20px;
-  margin: 8px 0;
-  display: inline-flex;
-  border: 1px solid white;
-  border-radius: 4px;
-  box-sizing: border-box;
-  color: white;
-  background-color: black;
+  text-align: left;
+  padding-top: 10px;
```

Comment:
`padding: 10px` you dont need to specify the top, bottom, etc if theyre all the same